site2/website-next/src/pages/download.js

```diff
@@ -327,6 +344,16 @@ export default function page(props) {
             <Translate>Older releases</Translate>
           </h2>
           <OldReleaseTable data={oldReleases}></OldReleaseTable>
+          <header className="postHeader mt-12">
+            <h1>
+              <Translate>Pulsar C++ Client</Translate>
+            </h1>
+            <div>
+              <p>The Link column contains C++ packages for different systems and architectures.</p>
+            </div>
+            <hr />
+          </header>
+          <ReleaseTable data={pulsarCppReleaseInfo}></ReleaseTable>
```

Comment:
You may provide a preview of this section.

Reply:
Updated.